src/modules/pass/hooks/useGetPassBenefits.ts

```diff
@@ -22,17 +24,25 @@ const useGetPassBenefits = ({
   const axios = useAxios(W3blockAPI.PASS);
   const { companyId: tenantId } = useCompanyConfig();
 
-  return usePrivateQuery([PixwayAPIRoutes.PASS_BENEFIT], () =>
-    axios.get<Response>(
-      PixwayAPIRoutes.PASS_BENEFIT.replace('{tenantId}', tenantId ?? ''),
-      {
-        params: {
-          tokenPassId: tokenPassId,
-          chainId: chainId && parseInt(chainId),
-          contractAddress: contractAddress,
-        },
-      }
-    )
+  return usePrivateQuery(
+    [PixwayAPIRoutes.PASS_BENEFIT],
+    () =>
+      axios.get<Response>(
```

Comment:
não precisa mudar nada , mas uma coisa que eu estou fazendo para saber depois oq eu to recebendo de resposta é primeiro eu to colocando o then para nao ter que depois fazer resposta.data.data.... e tambem eu to tipando o retorno da função que chama a request , para depois ficar mais facil de saber os possiveis campos